ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml

```diff
@@ -72,29 +72,13 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: make test-e2e
-      from: src
+      commands: pytest.sh -m "not ui"
+      from: insights-operator-tests
       resources:
         requests:
           cpu: 100m
     workflow: ipi-gcp
 - as: insights-operator-e2e-tests-periodic
-  cron: 0 0 * * *
-  steps:
-    cluster_profile: gcp
-    pre:
-    - chain: ipi-gcp-pre
-    - ref: insights-live
-    test:
-    - as: test
-      cli: latest
-      commands: make test-e2e
-      from: src
-      resources:
-        requests:
-          cpu: 100m
-    workflow: ipi-gcp
-- as: insights-operator-tests-periodic
```

Comment:
OK so this is removing the original integration test execution step right? I guess we can remove the integration tests from the IO repo once this is merged. WDYT?

Reply:
Right, this removes nightly test runs that use tests from IO repo. And yes, this directory can be removed https://github.com/openshift/insights-operator/tree/master/tests.